### ISSUE TITLE
regen: JSON-LD/Turtle prefix collides with regen: IRI scheme

### ISSUE THREAD

**Issue by aaronc** [regen-network/regen-registry-standards#76]
We use `regen:` as the prefix for `https://schema.regen.network#` in JSON-LD and Turtle and `regen:` is also the IRI scheme we use for the data module. It's sort of okay in Turtle because `regen:xyz` and `<regen:xyz>` are somewhat distinct but I think it's still confusing.

I suggest we use a different prefix such as `regenschema:`, `regenregistry:` or simply `r:` to differentiate. Our files could be re-written to use a different prefix without changing the actual content.

**Comment by blushi** [comment]
Good catch, it might be ok in turtle but it will cause some issues in JSON-LD so best to use a different prefix indeed. I'd prefer `regenschema:` or `r:`.

**Comment by blushi** [comment]
Let's update the regen-registry-standards mentions of `regen:` to `rs:` and in regen-web
